docs/usage/java.md

```diff
@@ -23,10 +23,52 @@ Renovate does not support:
 - Android projects that require extra configuration to run (e.g. setting the Android SDK)
 - Gradle versions prior to version 5.0.
 
+## Gradle Wrapper
+
+Renovate can update the [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) of a project.
+
+This includes the source declaration inside the `gradle/wrapper/gradle-wrapper.properties`
+as well as accompanied files such as `gradlew`, `gradlew.bat`, and `gradle/wrapper/gradle-wrapper.jar`.
```

Comment:
```suggestion
This includes the source declaration inside the `gradle/wrapper/gradle-wrapper.properties` as well as accompanied files such as `gradlew`, `gradlew.bat`, and `gradle/wrapper/gradle-wrapper.jar`.
```

We use the "one sentence per line" pattern here. This makes it easier to move lines around later on, or edit them, and have a easier Git diff.